rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2646,6 +2646,8 @@ def test_upgrade_db_45_to_46(user_data_dir: 'Path', messages_aggregator):
             'type, subtype) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
             history_event_bindings,
         )
+        # Ensure account_for_assets_movements is set before upgrade
+        write_cursor.execute("INSERT OR IGNORE INTO settings(name, value) VALUES ('account_for_assets_movements', 'True')")  # noqa: E501
```

Comment:
not going to stop the PR for this but it's best if instead you:

1. Manually edit the given user DB to just star by having this settings
2. Assert it's there before upgrade
3. AS you already do check after upgrade it's gone